modules/rosa-sts-creating-account-wide-sts-roles-and-policies.adoc

```diff
@@ -22,13 +22,13 @@ ifdef::quick-install[]
 * You have completed the AWS prerequisites for ROSA with STS.
 * You have available AWS service quotas.
 * You have enabled the ROSA service in the AWS Console.
-* You have installed and configured the latest ROSA CLI (`rosa`) on your installation host.
+* You have installed and configured the latest {product-title} (ROSA) CLI (`rosa`) on your installation host.
```

Comment:
as above, {product-title} has already been used so just ROSA CLI is fine here

Reply:
Maybe, "ROSA CLI (`rosa`)" here?